Naveen-gali/Yelp-App
# Search Bar

### What does this PR do?

- Adds TextInput Custom Component
- Adds Label Custom Component
- Adds SearchBar Component to home screen
- Adds Respective Stories.

### Related Issues

- closes #43 

### How to test

- Clone the Repository.
- Switch to feat/searchBar branch.
- Do yarn to install the dependencies.
- Do yarn run build-stage-ios for ios or yarn run build-stage-android for android.
- You Can see the SearchBar on Screen as shown in below images attached

### Screenshots

|Android ScreenShot           |  Ios ScreenShot|
|:-------------------------:|:-------------------------:|
![Screenshot_1683258710](https://user-images.githubusercontent.com/122969303/236374485-e661434f-1d92-4bd3-b322-537e20ec5a24.png) | ![Simulator Screen Shot - iPhone 14 - 2023-05-05 at 09 17 22](https://user-images.githubusercontent.com/122969303/236373923-a0d1f6a4-20b0-40e1-9310-32003c0b2985.png) |




### Checklist

- [x] I have addressed all the requirements.
- [x] I have self reviewed the PR.



### Packages Added (if any)
| Package | Reason |
| ------- | ------ |
|lodash.debounce| Lodash Package for Attaining Debounce        |
